bot/exts/events/advent_of_code/views/dayandstarview.py

```diff
@@ -64,12 +64,12 @@ async def day_select(self, select: discord.ui.Select, interaction: discord.Inter
         options=[discord.SelectOption(label=str(i)) for i in range(1, 3)],
         custom_id="star_select"
     )
-    async def star_select(self, select: discord.ui.Select, interaction: discord.Interaction) -> None:
+    async def star_select(self, _: discord.Interaction, select: discord.ui.Select) -> None:
         """Dropdown to choose either the first or the second star."""
         self.star = select.values[0]
 
     @discord.ui.button(label="Fetch", style=discord.ButtonStyle.blurple)
-    async def fetch(self, button: discord.ui.Button, interaction: discord.Interaction) -> None:
+    async def fetch(self, _: discord.ui.Button, interaction: discord.Interaction) -> None:
```

Comment:
Is `Button` still meant to come before `Interaction`?

Reply:
Good catch. This needs to be swapped, alongside the other uses of `@discord.ui.button`.

Reply:
Fixed :+1: